frontend/src/modules/Organizations/components/OrganizationTeams.js

```diff
@@ -44,7 +45,7 @@ import {
   OrganizationTeamInviteForm
 } from '../components';
 
-function TeamRow({ team, organization, fetchItems }) {
+function TeamRow({ team, permissions, organization, fetchItems }) {
```

Comment:
We should add permissions to the TeamRow properties in line 169

Reply:
Done